src/cli.ts

```diff
@@ -27,6 +31,11 @@ switch (cmd) {
   case "start":
     runStart();
     break;
+  case "--version": {
```

Comment:
I feel like we should consider adding some CLI lib. (Same applies to `create-waku`.)
I will look for something that fits well.

Reply:
Nodejs has official support for args. If you like it I could add the support

https://nodejs.org/dist/latest-v18.x/docs/api/util.html#utilparseargsconfig

Reply:
Oh, that looks good. I think we can use it meanwhile.